src/software/ai/navigator/navigator_test.cpp

```diff
@@ -365,6 +365,28 @@ TEST_F(NoPathNavigatorTest,
                                       testp1, testp2, testp3, final_speed));
 }
 
+TEST_F(NoPathNavigatorTest,
+       calculateTransitionSpeedBetweenSegments_tests_opposite_segments)
+{
+    Point testp1, testp2, testp3;
+    double final_speed;
+    // case 1
```

Comment:
Commenting `case 1` and `case 2` feels very vague, particularly when both tests are under the same test name.

Reply:
Do you want the comments removed?

Reply:
I'd prefer them to be expanded upon what the 1st and 2nd cases are if possible.

Reply:
They are two very similar cases that test opposite segments

Reply:
Perhaps we could just leave a comment indicating the difference / relation between the two cases to make this clear?

Reply:
ok done